sdk/python/jobs/automl-standalone-jobs/automl-distributed-classification-task-creditcard/classification-distributed-lightgbm.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "|**max_nodes**|The maximum number of nodes to use for `distributed` training. For classification/regression tasks, the minimum value that needs to be set is 4.|\r\n",
+    "\r\n",
+    "\r\n",
+    "##### set_training() parameters:\r\n",
+    "This is an optional configuration method to configure training parameters.\r\n",
+    "\r\n",
+    "|Property|Description|\r\n",
+    "|-|-|\r\n",
+    "|**allowed_training_algorithms**|A list of Regression algorithms to try out as base model for model training in an experiment. If it is omitted or set to None, then all supported algorithms are used during experiment|\r\n",
+    "|**enable_model_explainability**|A flag to turn on model explainability like feature importance, of best model evaluated by Automated ML system.|\r\n",
+    "|**training_mode**|[Experimental] The training mode to use. The possible values are-<ul>`distributed`- enables distributed training for supported algorithms.<ul>`non_distributed`- disables distributed training.<ul>`auto`- Currently, it is same as non_distributed. In future, this might change.<br><i>*Note*: This parameter is in public preview and may change in future.</i>|\r\n",
+    "\r\n"
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# General job parameters.\n",
+    "max_trials = 5\n",
+    "max_nodes = 6\n",
+    "exp_name = \"classification_task_distributed_lightgbm\"\n",
+    "target_column_name = \"Class\"\n",
+    "primary_metric = \"accuracy\""
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684844854618
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### 4.2 Create the AutoML classification job"
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Create the AutoML job with the related factory-function\n",
+    "classification_job = automl.classification(\n",
+    "    compute=cluster_name,\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=target_column_name,\n",
+    "    primary_metric=primary_metric,\n",
+    ")\n",
+    "\n",
+    "# Limits are all optional\n",
+    "classification_job.set_limits(\n",
+    "    timeout_minutes=120,\n",
+    "    trial_timeout_minutes=60,\n",
+    "    enable_early_termination=True,\n",
+    "    max_cores_per_trial=-1,\n",
+    "    max_trials=max_trials,\n",
+    "    max_concurrent_trials=4,\n",
```

Comment:
If we need to have a minimum of 4 max concurrent trials, we need to have max_nodes set to a minimum of 8. This is related to a comment I left above trying to understand why the min value is 4.

Reply:
Following discussion, I have removed the `max_concurrent_trials`. Thank you.